ls1intum/Artemis
# Fix exam summary page results

### Checklist

### Motivation and Context
Results were shown instantly on the exam summary page.

### Description
Moved resultsPublished from exercise.model back to the components.

### Steps for Testing
1. Create and participate in an exam.
2. When ended check for results and the view result buttons on three occasions:
2.1 No Release Date of Results
2.2 Too early for Release Date of Results
2.3 After Release Date of Results
